src/main/webapp/app/pages/curation/GeneListPage.tsx

```diff
@@ -67,7 +68,26 @@ const GeneListPage = (props: IGeneListPage) => {
       accessor: 'hugoSymbol',
       Header: 'Hugo Symbol',
       Cell(cell: { value: string }): any {
-        return <Link to={getCurationPageLink(cell.value, isGermline)}>{cell.value}</Link>;
+        const link = getCurationPageLink(cell.value, isGermline);
+
+        return (
+          <Link
+            to={undefined}
```

Comment:
What do you think about converting this to a Button that is styled like a link?

Reply:
I prefer to keep it as a link. That way any styling we make to the link element also apply here.